content/en/integrations/guide/reference-tables.md

```diff
@@ -88,6 +88,29 @@ Click **New Reference Table +**, then add a name, select AWS S3, fill out all fi
 
 [1]: https://app.datadoghq.com/account/settings#integrations/amazon-web-services
 [2]: https://docs.datadoghq.com/integrations/amazon_web_services/?tab=automaticcloudformation#installation
+{{% /tab %}}
+
+{{% tab "Azure storage" %}}
```

Comment:
right now we have the tabs `Manual upload`, `AWS S3 upload`, and `Azure storage`. we probably need to keep the word `upload` in the `Manual upload` tab, but maybe we can remove it from the S3 tab and just do `AWS S3` and `Azure storage` to keep them consistent? this would be similar to what logs archives does (though they don't have a manual option) https://docs.datadoghq.com/logs/log_configuration/archives/?tab=azurestorage#set-permissions